docs/utilities/sqs_batch.md

```diff
@@ -0,0 +1,460 @@
+---
+title: SQS Batch Processing (Deprecated)
+description: Utility
+---
+
```

Comment:
Here, add a big warning on the deprecation, a link to the batch module and a link to migration guide (don't know where we put that... in the new doc (add a chapter "Migrating from the old sqs batch module?")? or at the bottom of this one ?

Reply:
Done, and added a section at the bottom of `sqs_batch.md` - `Migrating to the Batch Library`